portfolio/src/main/java/com/google/sps/data/Contact.java

```diff
@@ -0,0 +1,17 @@
+
+package com.google.sps.data;
+
+public final class Contact {
```

Comment:
Please add a doc outlining the purpose of this class.

Reply:
do you mean comment?

Reply:
nit: It's a good practice to keep line length to 80 characters or less.

Reply:
Apologies that should have been docstring.